src/reanimated2/animation/util.ts

```diff
@@ -43,6 +44,26 @@ if (__DEV__ && IS_REDUCED_MOTION) {
   );
 }
 
+const SHOULD_BE_USE_WEB = shouldBeUseWeb();
+
+export function assertEasingIsWorklet(
+  easing: EasingFunction | EasingFunctionFactory
+) {
+  'worklet';
+
+  if (SHOULD_BE_USE_WEB) {
+    /* It is possible to run reanimated on web without plugin, so let's skip this check on web */
+    return true;
+  }
+  const isFunction: boolean = typeof easing === 'function';
+
+  if (isFunction && !isWorkletFunction(easing)) {
```

Comment:
Maybe we should move `typeof easing === 'function'` function to `isWorkletFunction`?